src/cgame/cg_draw.cpp

```diff
@@ -4633,6 +4634,10 @@ void CG_DrawMiscGamemodels(void) {
 }
 
 void CG_DrawCoronas() {
+  if (cg.demoPlayback && !ETJump::demoCompatibility->isCompatible("3.1.2")) {
```

Comment:
This method should probably accept a hardcoded struct instead so the parsing cost isn't paid on every invocation, e.g.
`isCompatible({3, 1, 2})`

Reply:
Assigned the results to static bool instead, I'd rather keep the input as a string for simplicity so that the compatibility checker is responsible for building the version struct.

Reply:
Actually, went with passing the struct rather than string.